CODEOWNERS

```diff
@@ -193,18 +181,83 @@ extensions/filters/http/oauth2 @rgs1 @derekargueta @snowp
 # Key Value store
 /*/extensions/key_value @alyssawilk @ryantheoptimist
-# Stateful session
-/*/extensions/http/stateful_session/cookie @wbpcode @dio
-# DNS Resolver
-/*/extensions/network/dns_resolver/cares @junr03 @yanavlasov
-/*/extensions/network/dns_resolver/apple @junr03 @yanavlasov
 # Config Validators
 /*/extensions/config/validators/minimum_clusters @adisuissa @htuch
+# DNS resolution
+/*/extensions/network/dns_resolver/cares @yanavlasov @mattklein123
+/*/extensions/network/dns_resolver/apple @yanavlasov @mattklein123
+# compression code
+/*/extensions/filters/http/decompressor @rojkov @kbaichoo
+/*/extensions/filters/http/gzip @gsagula @kbaichoo
+/*/extensions/filters/http/common/compressor @gsagula @rojkov @kbaichoo
+/*/extensions/filters/http/compressor @rojkov @kbaichoo
+/*/extensions/compression/brotli @rojkov @kbaichoo
+/*/extensions/compression/common @kbaichoo @rojkov
+/*/extensions/compression/gzip/decompressor @rojkov @kbaichoo
+/*/extensions/compression/gzip/compressor @rojkov @kbaichoo
+/*/extensions/compression/gzip/common @rojkov @kbaichoo
+# cel
+/*/extensions/access_loggers/filters/cel @kyessenov @douglas-reid @adisuissa
+# health cehck
+/*/extensions/filters/http/health_check @mattklein123 @adisuissa
+# lua
+/*/extensions/filters/http/lua @mattklein123 @wbpcode
+/*/extensions/filters/common/lua @mattklein123 @wbpcode
+# rbac
+/*/extensions/filters/network/rbac @yangminzhu @yanavlasov
+/*/extensions/filters/http/rbac @yangminzhu @yanavlasov
+/*/extensions/filters/common/rbac @yangminzhu @yanavlasov
+# tap
+/*/extensions/filters/http/tap @mattklein123 @daixiang0
+/*/extensions/common/tap @mattklein123 @daixiang0
+/*/extensions/transport_sockets/tap @mattklein123 @daixiang0
+# local rate limit
+/*/extensions/filters/network/local_ratelimit @mattklein123 @wbpcode
+
+# "Underowned" extensions (no backup reviewers, please volunter!)
+# Stateful session
+/*/extensions/http/stateful_session/cookie @wbpcode @UNOWNED
+/*/extensions/filters/http/stateful_session @wbpcode @UNOWNED
+# stats
+/*/extensions/stat_sinks/statsd @mattklein123 @UNOWNED
+/*/extensions/stat_sinks/common @mattklein123 @UNOWNED
+/*/extensions/stat_sinks/common/statsd @mattklein123 @UNOWNED
+# proxy protocol
+/*/extensions/filters/listener/proxy_protocol @ggreenway @UNOWNED
+# cors
+/*/extensions/filters/http/cors @wbpcode @UNOWNED
+# access loggers
+/*/extensions/access_loggers/grpc @wbpcode @UNOWNED
+/*/extensions/access_loggers/file @wbpcode @UNOWNED
+
+# "Unowned" extensions (currently without 1 maintainer or 2 owners)
+/*/extensions/formatter/req_without_query @UNOWNED @tsaarni
+# ext_authz
+/*/extensions/filters/common/ext_authz @esmet @gsagula @pradeepcrao @UNOWNED
+/*/extensions/filters/http/ext_authz @esmet @gsagula @pradeepcrao @UNOWNED
+/*/extensions/filters/network/ext_authz @esmet @gsagula @pradeepcrao @UNOWNED
+/*/extensions/filters/http/ip_tagging @UNOWNED @UNOWNED
+/*/extensions/filters/listener/original_dst @UNOWNED @UNOWNED
```

Comment:
This one is critical for istio, so one of me, @lambdai, @lizan should be the owners.

Reply:
Could do all 3 if @lizan Oks?

Reply:
We are big on ip_tagging -- I can help there.